### PR TITLE
Fixed Transaction Mapping and Sellers bugs

### PR DESCRIPTION
**Pull request by JustSamuel**
- Bug where credit invoice entries contained products from SubTransactions with different sellers

- Bug where transaction report reported products being sold from subTransactions from different sellers.

### CODE REVIEW

**Review by Yoronex — CHANGES_REQUESTED** [review]

File: src/controller/request/invoice-request.ts [hunk 1]
```diff
@@ -62,11 +62,12 @@ export interface CreateInvoiceParams extends BaseInvoice {
  * @property {integer} forId.required - The recipient of the Invoice.
  * @property {integer} byId - The creator of the Invoice, defaults to the ID of the requester.
  * @property {string} addressee.required - Name of the addressed.
- * @property {string} description.required - The description of the invoice.
```
Comment (CodeNamedRobin) [inline src/controller/request/invoice-request.ts:65]: Wrong string.

File: src/controller/request/invoice-request.ts
Comment (CodeNamedRobin) [inline src/controller/request/invoice-request.ts:1]: First changed string should be unchanged.

File: test/unit/service/invoice-service.ts
Comment (CodeNamedRobin) [inline test/unit/service/invoice-service.ts:1]: Is correct, but I smell a ESLint fixer being used so Samuel gets to trek a bak.

**Review by CodeNamedRobin — CHANGES_REQUESTED** [review]

**Review by CodeNamedRobin — APPROVED** [review]
LGTM!

**Pull request merged**